lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -1162,7 +1170,7 @@ void LocalEnforcer::update_charging_credits(
         session->get_session_info(info);
         for (const auto& rule : info.gy_dynamic_rules) {
           PolicyRule dy_rule;
-          auto& uc = session_update[imsi][session->get_session_id()];
+          uc = session_update[imsi][session->get_session_id()];
```

Comment:
do we need to redefine it here? does the session ID change from line 1160?

Reply:
oh right. This was impacted by a merge conflict. I may have missed it